pycbc/types/timeseries.py

```diff
@@ -147,6 +147,21 @@ def get_sample_rate(self):
         return int(1.0/self.delta_t)
     sample_rate = property(get_sample_rate,
                            doc="The sample rate of the time series.")
+
+    def select_time(self, start, end):
```

Comment:
I might suggest making this select_time_slice, or select_time_window, or something else to suggest a *range* of times, not a single time.